mlflow/utils/conda.py

```diff
@@ -149,6 +175,7 @@ def get_or_create_conda_env(conda_env_path, env_id=None, capture_output=False):
                         "--file",
                         conda_env_path,
                     ],
+                    env=additional_env,
```

Comment:
```suggestion
                    extra_env=additional_env,
```

I think we should use `extra_env` here.